nixos/modules/services/networking/firefox/sync-server.nix

```diff
@@ -124,7 +124,13 @@ in
       description = "Firefox Sync Server";
       wantedBy = [ "multi-user.target" ];
       path = [ pkgs.pythonPackages.pasteScript pkgs.coreutils ];
-      environment.PYTHONPATH = "${pkgs.pythonPackages.syncserver}/lib/${pkgs.pythonPackages.python.libPrefix}/site-packages";
+      environment.PYTHONPATH = let
+          penv = pkgs.python.buildEnv.override {
+            extraLibs = [ pkgs.pythonPackages.syncserver ];
+            ignoreCollisions = true;
+          };
+        in "${penv}/${pkgs.python.sitePackages}/";
```

Comment:
This works and might be a bit cleaner: https://github.com/Nadrieril/nixpkgs/commit/a0cc46de4c8cc3000a164696f058b923ecb1508a